changelogs/2.11.6.md

```diff
@@ -0,0 +1,158 @@
+indexes, but **ever had it before**, it is still unsafe. A `double` index
+altered to `number` or `scalar` would remain broken. It is **very important that
+the users rebuild all the vinyl indexes that have or ever had the `double` field
+type in them**. What is worse, just an alter `double` -> `number` **won't
+work**. The old index must be dropped and a new one must be created. In any
+order.
+
+See more details in gh-9965 and [gh-11017][gh-11017].
+
+[gh-11017]: https://github.com/orgs/tarantool/discussions/11017
+
+## Overview
+
+2.11.6 is the next stable release in the [long-term support (LTS)
+version][release_policy] 2.11.x release series.
+
+The label "stable" means there are 2.11.x-based applications running in
+production for quite a while without known crashes, incorrect results or
+other showstopper bugs.
+
+This release resolves 32 issues since the 2.11.5 version.
+
+[release_policy]: https://www.tarantool.io/en/doc/latest/release/policy/
+[issues]: https://github.com/tarantool/tarantool/issues
+
+## Compatibility
+
+Tarantool 2.11.x is backward compatible with Tarantool 1.10.x in binary data
+layout, client-server protocol, and replication protocol.
+
+Please [upgrade][upgrade] using the `box.schema.upgrade()` procedure to unlock
+all the new features of the 2.x series.
+
+[upgrade]: https://www.tarantool.io/en/doc/latest/book/admin/upgrades/
+
+## Bugs fixed
+
+### Core
+
+* Fixed a bug when a server could crash if a client sent an IPROTO replication
+  request without waiting for pending requests to complete (gh-10155).
+* Fixed a bug when the `sequence:reset()` call result was not recovered
+  after the server restart (gh-9871).
+* Fixed several bugs in the numbers comparison and hashing in tuple keys. It
+  allowed to insert the same key multiple times into a unique index, and
+  sometimes wouldn't allow to find an existing key in an index. Could happen
+  when numbers were encoded in MessagePack suboptimally and when the `double`
+  field type was used (gh-9965).
+* Fixed the issue with hanging write operations forever triggered by heavy
+  write load and WAL writing failures on cascade rollback (gh-11081).
+
+### Memtx
+
+* Fixed a crash when memtx MVCC tried to abort an already committed
+  non-memtx transaction if it used a system space view or tried to
+  perform a DDL operation (gh-10715).
+* Fixed a bug when Tarantool with memtx MVCC enabled was aborted on
+  workload with many `index:get()` operations reading nothing (gh-11022).
+
+### Vinyl
+
+* Fixed a bug when a deleted tuple wasn't purged from a secondary index data
+  stored on disk in case it was updated more than once in the same transaction.
+  The bug couldn't result in inconsistent query results, but it could lead to
+  performance degradation and increased disk usage (gh-10820, gh-10822).
+* Fixed a bug when joining a new replica to a master instance that experiences
+  a heavy write load would severely degrade the master instance performance.
```

Comment:
```suggestion
  a heavy write load severely degrades the master instance performance.
```

Reply:
Fixed.

Reply:
Sent for `3.2.2` in #11197. Can't send it for 3.3.2 and 3.4.0, because the change is already released in 3.3.0.